<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TreeView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioShowAll">
            <property name="text">
             <string>显示目录文件</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QRadioButton" name="radioShowOnlyDir">
            <property name="text">
             <string>只显示目录</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkBoxEnableFilter">
            <property name="text">
             <string>文件名过滤</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnApplyFilters">
            <property name="text">
             <string>应用</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/images/icons/828.bmp</normaloff>:/images/icons/828.bmp</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QTreeView" name="treeView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>ListView</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListView" name="listView"/>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>TableView</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="labFileName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>53</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>FileName</string>
       </property>
      </widget>
      <widget class="QLabel" name="labFileFileSize">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>53</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>0 KB</string>
       </property>
      </widget>
      <widget class="QLabel" name="labType">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>File Folder</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkIsDir">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>20</y>
         <width>80</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>节点是目录</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSetRootDir"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actSetRootDir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/fold.bmp</normaloff>:/images/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>设置根目录</string>
   </property>
   <property name="toolTip">
    <string>设置根目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
